src/ui/units/wizard/components/Dialogs/DialogField/DialogField.tsx

```diff
@@ -267,6 +271,12 @@ class DialogField extends React.PureComponent<DialogFieldInnerProps, DialogField
                 !Number.isNaN((formatting as CommonNumberFormattingOptions).precision);
         }
 
+        const modalBody = this.renderModalBody();
+        if (!modalBody) {
+            this.props.onCancel();
```

Comment:
Can you add a comment here why do I need to call cancel? So as not to be deleted accidentally later